src/library/Structs.sol

```diff
@@ -1,34 +1,71 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.23;
 
+/**
+ * @dev Enumeration for different levels of an organization.
+ *      - FirstLevel: Represents a first-level organization.
+ *      - SecondLevel: Represents a second-level organization.
+ */
 enum OrganizationLevel {
     FirstLevel,
     SecondLevel
 }
 
+/**
+ * @dev Enumeration for different labels that can be assigned to objects.
+ *      - Project: Represents a project.
+ *      - Proposal: Represents a proposal.
+ *      - Milestone: Represents a milestone.
+ */
 enum Label {
     Project,
     Proposal,
     Milestone
 }
 
+/**
+ * @dev Structure representing a block in the system.
+ * @param user The identity of the user who created the block.
+ * @param timestamp The timestamp when the block was created.
+ * @param content The content of the block.
+ */
 struct Block {
     Identity user;
     uint64 timestamp;
     string content;
 }
 
+/**
+ * @dev Structure representing a public key.
+ * @param key The address of the public key.
+ * @param signature_verifier The address of the signature verifier.
+ */
 struct PublicKey {
     address key;
     address signature_verifier;
 }
 
+/**
+ * @dev Structure representing an identity.
+ * @param handle The handle associated with the identity.
+ * @param public_keys An array of public keys associated with the identity.
+ * @param recovery_addresses An array of two recovery addresses.
+ */
 struct Identity {
     bytes32 handle;
     PublicKey[] public_keys;
     address[2] recovery_addresses;
 }
 
+/**
+ * @dev Structure representing an organization.
+ * @param offchain_uri The URI for off-chain data associated with the organization.
+ * @param level The level of the organization.
+ * @param owner The identity of the owner of the organization.
+ * @param managed_identities An array of identities managed by the organization.
+ * @param association The ID of the associated parent organization, if any.
+ * @param is_verified A boolean indicating whether the organization is verified.
+ */
 struct Organization {
     string offchain_uri;
```

Comment:
the offchain URI can be decided on it,s basically packing ORG info that may be too large to put onchain. the contract only stores the uri.
it can also be an empty string if the org has no off-chain URI